src/software/parameter/config/robot_diagnostics_main_command_line_args.yaml

```diff
@@ -0,0 +1,12 @@
+RobotDiagnosticsMainCommandLineArgs:
+    backend:
```

Comment:
Should we make it a rule that command-line args be `snake_case` instead of `CamelCase` like other dynamic parameters? If so, we should probably document this somewhere.

Reply:
Yeah agreed, but this doc currently doesn't exist, and the new dynamic parameter system is in the works

I've made a ticket to capture your comment + actually adding dynamic parameter docs here:
https://github.com/UBC-Thunderbots/Software/issues/1738

It should be unblocked as soon as the new system goes in

Reply:
Ok sounds like a plan 👍